### ISSUE TITLE
Allow us to customize Shortcuts

### ISSUE THREAD

**Issue by OverboyDev** [Scirra/Construct-feature-requests#12]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

Allow us to customize Shortcuts

### Possible workarounds or alternatives

Need to use external software to remap keyboard, but even doing so it is very tedious and limite

### Proposed solution

As it may be complicated to support shortcut customisation for some specific things (CTRL + C for example) because they're used in so many place and for different features, maybe the whole "General Shortcuts" category (or almost) could be unavailable to edit/grayed out, same for some very specific other shortcuts.

An other benefit is that once the Shortcut Editor is integrated in C3, the only thing Scirra would need to add some QOL stuff would be to add a new shortcut action. (no need to worry about context menu or if that key is available for a new shortcut). The user you would need this new QOL would just map it to the desired shortcut.

This way a lot of feature request could just be "please add shortcut support for this thing".

### Why is this idea important?

For a lot of us, C3 is our main tool we use 5-12 hours per day, so allowing us to customize shortcuts as we wants would allow us to save several hours of work per week. Most of the tool we used allow to remap shortcuts, it's a shame our main tool doesn't.

**Comment by xtrad** [comment]
Yes, I'm struggling at the moment I switch Photoshop & Construct about Ctrl+MouseScroll, Alt+MouseScroll..
It is very painful
